Tests/subset/svg_test.py

```diff
@@ -0,0 +1,455 @@
+from string import ascii_letters
+import textwrap
+
+from fontTools.misc.testTools import getXML
+from fontTools import subset
+from fontTools.fontBuilder import FontBuilder
+from fontTools.pens.ttGlyphPen import TTGlyphPen
+from fontTools.ttLib import TTFont, newTable
+from fontTools.subset.svg import NAMESPACES
+
+import pytest
+
+etree = pytest.importorskip("lxml.etree")
+
+
+@pytest.fixture
+def empty_svg_font():
+    glyph_order = [".notdef"] + list(ascii_letters)
+
+    pen = TTGlyphPen(glyphSet=None)
+    pen.moveTo((0, 0))
+    pen.lineTo((0, 500))
+    pen.lineTo((500, 500))
+    pen.lineTo((500, 0))
+    pen.closePath()
+    glyph = pen.glyph()
+    glyphs = {g: glyph for g in glyph_order}
+
+    fb = FontBuilder(unitsPerEm=1024, isTTF=True)
+    fb.setupGlyphOrder(glyph_order)
+    fb.setupCharacterMap({ord(c): c for c in ascii_letters})
+    fb.setupGlyf(glyphs)
+    fb.setupHorizontalMetrics({g: (500, 0) for g in glyph_order})
+    fb.setupHorizontalHeader()
+    fb.setupOS2()
+    fb.setupPost()
+    fb.setupNameTable({"familyName": "TestSVG", "styleName": "Regular"})
+
+    svg_table = newTable("SVG ")
+    svg_table.docList = []
+    fb.font["SVG "] = svg_table
+
+    return fb.font
+
+
+def new_svg(**attrs):
+    return etree.Element("svg", {"xmlns": NAMESPACES["svg"], **attrs})
+
+
+def test_subset_svg_simple(empty_svg_font, tmp_path):
+    # 'simple' as in one glyph per svg doc
```

Comment:
test_subset_otsvg_single_glyph_per_svg_doc (or some such) and drop the comment? - intent being clearer notion of what broke when a failure reports